demo/meeting/src/containers/VideoStreamMetrics/index.tsx

```diff
@@ -0,0 +1,191 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+
+import {
+  useMediaStreamMetrics,
+  useAudioVideo,
+  PopOverHeader
+} from 'amazon-chime-sdk-component-library-react';
+
+import MediaStatsList from '../../components/MediaStatsList/index';
+import MetricItem from '../../components/MediaStatsList/MetricItem';
+import { StyledMediaMetricsWrapper } from '../../components/MediaStatsList/Styled';
+
+function isValidMetric(metric: any) {
+  return typeof metric === 'number' && !Number.isNaN(metric);
+}
+
+interface Props {
+  /** The Chime attendee ID */
+  attendeeId: string;
+}
+
+export const VideoStreamMetrics: React.FC<Props> = ({ attendeeId }) => {
+  const audioVideo = useAudioVideo();
+  const { videoStreamMetrics } = useMediaStreamMetrics();
+  const streamMetric = videoStreamMetrics[attendeeId];
+  const ssrcArray = streamMetric ? Object.keys(streamMetric) : [];
+  const showMetric =
+    audioVideo && attendeeId && streamMetric && ssrcArray.length !== 0;
+  return (
+    <StyledMediaMetricsWrapper>
+      {showMetric && (
+        <>
+          <PopOverHeader
+            title={"Video Statistics"}
+          />
+          <MediaStatsList>
+            <MetricItem
+              metricName="Bit rate (kbps)"
+              metricValues={
+                ssrcArray.map((ssrc) => {
+                  return isValidMetric(streamMetric[ssrc].videoDownstreamBitrate)
+                    ? Math.trunc(streamMetric[ssrc].videoDownstreamBitrate/1000).toString()
+                    : '';
+                })
+              }
+            />
+            <MetricItem
+              metricName="Packet Loss"
+              metricValues={
+                ssrcArray.map((ssrc) => {
+                  return isValidMetric(streamMetric[ssrc].videoDownstreamPacketLossPercent)
+                    ? Math.trunc(streamMetric[ssrc].videoDownstreamPacketLossPercent).toString()
+                    : '';
+                })
+              }
+            />
+            <MetricItem
+              metricName="Frame Rate"
+              metricValues={
+                ssrcArray.map((ssrc) => {
+                  return isValidMetric(streamMetric[ssrc].videoDownstreamFramesDecodedPerSecond)
+                    ? streamMetric[
+                        ssrc
```

Comment:
Same comment for the indentation, can we keep this in single line? Reads better :)

Reply:
I tried to make them in oneline but prettier formatter will separate them for me.